shell/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -276,7 +276,9 @@ OffScreenRenderWidgetHostView::~OffScreenRenderWidgetHostView() {
   // Marking the DelegatedFrameHost as removed from the window hierarchy is
   // necessary to remove all connections to its old ui::Compositor.
   if (is_showing_)
-    delegated_frame_host_->WasHidden();
+    // TODO(deermichel): correct or kOther?
+    delegated_frame_host_->WasHidden(
+        content::DelegatedFrameHost::HiddenCause::kOccluded);
```

Comment:
Should be `kOther` the scenario chromium CL fixes seems not useful for OSR case, /cc @brenca for review on this change.